src/vulkan/graphics_pipeline.cc

```diff
@@ -819,7 +815,19 @@ Result GraphicsPipeline::ClearBuffer(const VkClearValue& clear_value,
 
   DeactivateRenderPassIfNeeded();
 
-  return frame_->CopyColorImagesToHost(command_.get());
+  r = frame_->CopyColorImagesToHost(command_.get());
```

Comment:
Is this change related to pipeline removal from probe commands?

Reply:
ditto